doc/api/net.md

```diff
@@ -1028,6 +1028,23 @@ written out, which may not be immediately.
 See `Writable` stream [`write()`][stream_writable_write] method for more
 information.
 
+### `socket.readyState`
+<!-- YAML
+added: v0.5.0
+-->
+
+* {string}
+
+This property represents the state of the connection as a string.
+
+If the stream is connecting `socket.readyState` is `opening`
+
+If the stream is readable and writable it is `open`
+
+If the stream is readable and not writable it is `readOnly`
+
+If the stream is not readable and writable it is `writeOnly`
```

Comment:
```suggestion
- If the stream is connecting `socket.readyState` is `opening`.
- If the stream is readable and writable, it is `open`.
- If the stream is readable and not writable, it is `readOnly`.
- If the stream is not readable and writable, it is `writeOnly`.
```

(I don’t really care about turning this into a list, but ending the sentences with periods is nice. :slightly_smiling_face: )